backend/private-graph/graph/resolver.go

```diff
@@ -76,6 +76,8 @@ const ErrorGroupLookbackDays = 7
 const SessionActiveMetricName = "sessionActiveLength"
 const SessionProcessedMetricName = "sessionProcessed"
 
+const AuthorizationError = "403 - AuthorizationError"
```

Comment:
nit: i think you could make this a `var AuthorizationError = errors.New("403 - AuthorizationError")` so that you could just `return nil, AuthorizationError` below

Reply:
Good idea. 
Fixed in d693f80befb5b3e33b1939617bafc2ae9c81583f